lib/include/ultrahdr/jpegr.h

```diff
@@ -29,19 +29,19 @@
 namespace ultrahdr {
 
 // Default configurations
-// Map is quarter res / sixteenth size
-static const size_t kMapDimensionScaleFactorDefault = 4;
+// gainmap image downscale factor
+static const size_t kMapDimensionScaleFactorDefault = 1;
```

Comment:
Actually can we keep this and `kUseMultiChannelGainMapDefault` to the old values? This library is also used by Android framework and Android is not ready for moving to this this high bar yet. On the other side since we have updated the values in the app so the app will have the new default values.

Reply:
I mean can we only keep the original configuration `kMapDimensionScaleFactorDefault = 4`? The app's default is 1 and is passing it to here and will bypass 4